client/Assets/Scripts/UI/UIModelManager.cs

```diff
@@ -61,4 +67,32 @@ IEnumerator AnimateCharacter(GameObject modelClone)
             yield return new WaitForSeconds(ANIMATION_INTERVAL);
         }
     }
+
+    public void ShowEndGameCharacterAnimation()
+    {
+        if (Utils.GetCharacter(GameServerConnectionManager.Instance.playerId))
+        {
+            bool isWinner = GameServerConnectionManager
+                .Instance
+                .PlayerIsWinner(GameServerConnectionManager.Instance.playerId);
+            string animationName = isWinner ? "Victory" : "Defeat";
+            if (modelAnimator.parameterCount > 0)
+            {
+                HandleAnimation(animationName);
+            }
+        }
+    }
+
+    public void HandleAnimation(string animationName)
+    {
+        bool hasAnimationParameter = modelAnimator.parameters.ToList().Find(anim => anim.name == animationName) != null;
```

Comment:
i think using `Any` is cleaner than comparing with null right?

```suggestion
        bool hasAnimationParameter = modelAnimator.parameters.ToList().Any(anim => anim.name == animationName);
```